scripts/typescript/src/bcdb/bcdb.ts

```diff
@@ -0,0 +1,352 @@
+/**
+ * The purpose of this script is to load  bdcb.xlsx file and export it as a JSON file.
+ *
+ *
+ * Documentation reference on XSLX: https://docs.sheetjs.com/docs
+ *
+ * This script does not import all the sheets and columns.
+ * See link to google drive files above (column_meanings.xlsx)
```

Comment:
I think I should include the column_meaning.xlsx to the repo, since there is the xlsx for the data too. I cannot reference the drive (it won't be accessible by the public, and we don't want it).